tests/smoke/support/api/recevedFolder.ts

```diff
@@ -23,3 +22,18 @@ export const setReceivedFolder = async ({
 
   checkResponseStatus(response, 'Failed creating received folder')
 }
+
+export const disablesAutoAccept = async ({ admin }: { admin: User }): Promise<void> => {
```

Comment:
i would love to have something like `autoAccept(true)` instead of `disablesAutoAccept (...)`, then we're able to turn it on and off depending on usecase